metricbeat/etc/beat.yml

```diff
@@ -55,4 +55,5 @@ metricbeat.modules:
   period: 2s
   processes: ['.*']
 
-
+  # if true, exports the CPU usage in ticks, together with the percentage values
+  cpu_usage_in_ticks: false
```

Comment:
I would call the option just `ticks` as it is already for the metricset cpu and core.